nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/HandleHttpRequest.java

```diff
@@ -446,6 +447,7 @@ private SslContextFactory createSslFactory(final SSLContextService sslService, f
 
         sslFactory.setNeedClientAuth(needClientAuth);
         sslFactory.setWantClientAuth(wantClientAuth);
+        sslFactory.setProtocol(sslService.getSslAlgorithm());
```

Comment:
Yep, the exact same problem. Once resolved on https://github.com/apache/nifi/pull/1986, i'll do the same here.